src/runtime/manifest-parser.pegjs

```diff
@@ -1972,9 +1969,12 @@ UnitName
     return unit+'s';
   }
 
+TypeIdentifier = typeIdentifier:('n'/'i'/'l'/'f'/'d') &[^a-zA-Z0-9_] {
```

Comment:
Could you add a comment for this one? I don't really understand what it is.

Reply:
Updated, any feedback?

Reply:
That helps, thanks!